uber/templates/registration/new_base.html

```diff
@@ -17,10 +17,6 @@
         }, 900000);
     {% endif %}
 
-    {% if c.REG_USES_BARCODES %}
```

Comment:
It works differently, you don’t need to set the detect_barcode_scan anymore, which is all this did. The reg page now supports both QR codes on the search bar, and badge nums on the checkin dialog. See above 🤪

Reply:
This page doesn't extend or even have anything to do with `registration/index.html` -- it won't have the barcode scanning detection included just because the other registration page has it. It needs to be included separately, i.e., `{% include 'barcode_client.html' %}`.

Probably confusing, since both pages are used for check-in, but from a code perspective they're pretty much unrelated. 😓

Reply:
Okay, I’ll fix this tomorrow.

Reply:
On an unrelated note, we gotta get rid of all these *_base.html files.